src/sonic-host-services/tests/hostcfgd/hostcfgd_test.py

```diff
@@ -1,6 +1,6 @@
 import os
 import sys
-import swsssdk
+import swsscommon
```

Comment:
just minor cosmetic changes, same as the changes in other files, can you change this to "from swsscomm import swsscommon"?

Reply:
I submit another PR on master to solve your comments: [https://github.com/Azure/sonic-buildimage/pull/8515](https://github.com/Azure/sonic-buildimage/pull/8515).

Keep this PR as simple as backport.